lib/client.go

```diff
@@ -395,6 +413,17 @@ func GetWorkspaceVar(organization, wsName, key string) (*Var, error) {
 
 // GetVarsFromWorkspace returns a list of Terraform variables for a given workspace
 func GetVarsFromWorkspace(organization, workspaceName string) ([]Var, error) {
+	orgExists, err := OrganizationExists(organization)
+	if err != nil {
+		return []Var{}, fmt.Errorf("GetVarsFromWorkspace: organization is required")
```

Comment:
The usual way of handling errors in Go is to chain the messages like this:
```suggestion
		return []Var{}, fmt.Errorf("GetVarsFromWorkspace: organization is required: %w", err)
```